forc/src/utils/defaults.rs

```diff
@@ -33,9 +33,10 @@ version = "0.1.0"
 [dependencies]
 fuel-gql-client = {{ version = "0.4", default-features = false }}
 fuel-tx = "0.6"
-fuels-abigen-macro = "0.6"
-fuels-contract = "0.6"
-fuels-core = "0.6"
+fuels-abigen-macro = "0.7"
+fuels-contract = "0.7"
+fuels-core = "0.7"
+fuels-signers = "0.7"
```

Comment:
v0.8.0 should have fixed the issues
```suggestion
fuels-abigen-macro = "0.8"
fuels-contract = "0.8"
fuels-core = "0.8"
fuels-signers = "0.8"
```